sdk/src/main/java/ch/admin/bag/covidcertificate/eval/utils/RecoveryEntryUtils.kt

```diff
@@ -33,10 +33,14 @@ fun RecoveryEntry.dateFormattedOfFirstPostiveResult(dateFormatter: DateTimeForma
 	}
 }
 
-fun RecoveryEntry.getRecoveryCountry(): String {
+fun RecoveryEntry.getRecoveryCountry(deviceLang: String): String {
 	return try {
```

Comment:
This whole country string logic is the same for recoveries, tests and vaccinations. Should we extract the logic into a separate Utils class and only pass in the `co` from the *Entry?